### PR TITLE
Replacing Jarsigner with Apksigner

### PR DESCRIPTION
**Pull request by Dado1513**
- Replacing Jarsigner with Apksigner
- Update README.md with new requirements
- Add Test case for Apksigner

### Description

Replacing JarSigner with Apksigner to sign apps, [issue 63](https://github.com/ClaudiuGeorgiu/Obfuscapk/issues/63)

### Type of change

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [x] Other (e.g., refactoring, documentation, etc.)

### Checklist

* [x] I have read the [contributing](https://github.com/ClaudiuGeorgiu/Obfuscapk/blob/master/docs/CONTRIBUTING.md) guidelines
* [x] I have performed a self-review of my own code (added comments, corrected misspellings etc.)
* [x] My code follows the style guidelines of this project
* [x] I have made corresponding changes to the documentation (if appropriate)
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing tests pass locally with my changes

P.S.: sembrava impossibile, ma alla fine ce l'ho fatta.

### CODE REVIEW

**Comment by ClaudiuGeorgiu** [comment]
image

**Comment by ClaudiuGeorgiu** [comment]
Come on, @Dado1513, you can do it (or maybe not)!

**Comment by Dado1513** [comment]
image

**Comment by ClaudiuGeorgiu** [comment]
image

File: README.md [hunk 1]
```diff
@@ -173,10 +174,10 @@ Copyright (C) 2009 The Android Open Source Project
 ...
 ```
 
-To install and use `apktool` you need a recent version of Java, which should also have
-`jarsigner` bundled. `zipalign` is included in the Android SDK. The location of the
+To install and use `apktool` you need a recent version of Java. 
+`zipalign` and `apksigner` is included in the Android SDK. The location of the
```
Comment (ClaudiuGeorgiu) [inline README.md:178]: `zipalign` and `apksigner` are included

**Review by ClaudiuGeorgiu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Dado1513) [reply README.md]: I did it

Comment (ClaudiuGeorgiu) [reply README.md]: image

**Pull request merged**